NuGet.config

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <clear />
+    <add key="dotnet-core" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
+    <add key="extensions" value="https://dotnetfeed.blob.core.windows.net/aspnet-extensions/index.json" />
+    <add key="entityframeworkcore" value="https://dotnetfeed.blob.core.windows.net/aspnet-entityframeworkcore/index.json" />
+    <add key="NuGet.org" value="https://api.nuget.org/v3/index.json" />
+  </packageSources>
```

Comment:
Might as well add in `npgsql-stable` and `npgsql-unstable` while we're at it?

Reply:
I generally refrain from adding feeds which aren't being used, to prevent the possibility of accidentally taking versions I don't want etc. At the moment we're only using stable version of Npgsql from nuget.org so there's no real need...